src/cljs/puppetlabs/puppetserver/dashboard/dashboard/production.cljs

```diff
@@ -0,0 +1,4 @@
+(ns puppetlabs.puppetserver.dashboard.production
```

Comment:
Thanks @rlinehan.  I bumped this and the other cljs files up a level in e698ecb since this appears to more closely match what we did in PE Puppet Server.